build.gradle.kts

```diff
@@ -126,6 +126,7 @@ dependencies {
     // langchain4j integrations
     implementation("dev.langchain4j:langchain4j-open-ai:0.32.0")
     implementation("dev.langchain4j:langchain4j-ollama:0.32.0")
+    implementation("dev.langchain4j:langchain4j-qianfan:0.32.0")
```

Comment:
Please add a comment that this is for ERNIE model, since it's not immediately recognizable from the dependency name.